docs/site/downloads/cldr-48.md

```diff
@@ -339,20 +318,17 @@ For example: El Aaiun → El Aaiún; Casey → Casey Station; Hovd Time → Khov
 ### V49 advance warnings
 
 The following changes are planned for CLDR 49. Please plan accordingly to avoid disruption.
-- [CLDR-18303][] H24 will be deprecated. If it is encountered, it will have H23 behavior. There is no known intentional usage of H24. If you have a current need for H24 instead of H23, please comment on [CLDR-18303][].
-- The default week numbering changes to ISO instead being based on the calendar week starting in CLDR 48 [CLDR-18275]. The calendar week will be more clearly targeted at matching usage in displayed month calendars.
+- H24 will be deprecated. If it is encountered, it will have H23 behavior. There is no known intentional usage of H24. If you have a current need for H24 instead of H23, please comment on [CLDR-18303][].
+- The default week numbering system changes to follow ISO, instead of being based on the calendar week, starting in CLDR 48. The calendar week will be more clearly targeted at matching usage in displayed month calendars. [CLDR-18275][]
```

Comment:
```suggestion
- As of CLDR 49, the default week numbering system will change to follow ISO (where weeks are numbered based on Thursday), instead of being based on the start of the calendar week. The calendar week will be more clearly targeted at matching usage in displayed month calendars. [CLDR-18275][]
```